### PR TITLE
Text utilities and lists

### PR DESCRIPTION
**Pull request by Manju-sree**
Added Text Utilities and Lists Components.
Text Utilities
- headings
- small text
- gray text
- center text

Lists
- Spaced (normal lists)
- Stacked (Notification Page)

### CODE REVIEW

File: Components/Lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+.list-style-none {
+    list-style: none;
+}
+
+
+/* stacked list for notification */
+
+.stacked-list-container {
+    border-radius: 0.3rem;
+    background-color: #f7f5f5;
```
Comment (rushikesh3299) [inline Components/Lists/lists.css:32]: Try to use variables for color & bg

**Review by rushikesh3299 — COMMENTED** [review]
Code is well organized & classnames are also descriptive. Looks good to me. You can merge

**Review by ttahm3d — APPROVED** [review]
Looks good. Please go ahead and merge.

File: Components/Lists/lists.css (hunk 1 above)
Comment (Manju-sree) [reply Components/Lists/lists.css:32]: please approve to merge it

**Review by rushikesh3299 — APPROVED** [review]
Good to merge

**Pull request merged**